pkgs/applications/audio/distrho/default.nix

```diff
@@ -2,12 +2,12 @@
 , libxslt, lv2, pkgconfig, premake3, xorg, ladspa-sdk }:
 
 stdenv.mkDerivation rec {
-  name = "distrho-ports-unstable-2017-10-10";
+  name = "distrho-ports-unstable-2018-01-01";
 
   src = fetchgit {
```

Comment:
This PR itself is fine but I would appreciate it if we could change this to use `fetchFromGitHub`.

Reply:
OK, done.